src/Adapter/MSTest.CoreAdapter/Execution/TestAssemblySettingsProvider.cs

```diff
@@ -56,6 +56,13 @@ internal TestAssemblySettings GetSettings(string source)
 
             testAssemblySettings.CanParallelizeAssembly = !this.reflectHelper.IsDoNotParallelizeSet(testAssembly);
 
+            var classCleanupSequencingAttribute = this.reflectHelper.GetClassCleanupAttribute(testAssembly);
```

Comment:
Is this naming based on some existing settings we have? I think we should choose between Sequencing and Lifecycle if there is no existing example of this and in that case Lifecycle seems easier to grasp.

Or possibly ClassCleanupExecution?